lib/server/ServerRenderer.js

```diff
@@ -72,24 +71,33 @@ function makeClientAppStartScript(clientAppRequirePath, environmentConfig, boots
         "});";
 }
 
-function makeBootstrappingLoader(scriptBody, clientAppUrl, isAsync) {
-    var loadAsync = isAsync ? "s.async = true;\n" : "";
```

Comment:
We actually might want to use defer so the script loads when the page has finished parsing